src/snovault/elasticsearch/indexer.py

```diff
@@ -293,13 +305,77 @@ def get_current_xmin(request):
     xmin = query.scalar()  # lowest xid that is still in progress
     return xmin
 
+
 class Indexer(object):
     def __init__(self, registry):
         self.es = registry[ELASTIC_SEARCH]
         self.esstorage = registry[STORAGE]
         self.index = registry.settings['snovault.elasticsearch.index']
+        self.queue_server = None
+        self.queue_client = None
+        queue_type = registry.settings.get('queue_type', None)
+        is_queue_server = asbool(registry.settings.get('queue_server'))
+        is_queue_client = asbool(registry.settings.get('queue_client'))
+        queue_client_processes = int(
+            registry.settings.get('queue_client_processes', 1)
+
+        )
+        queue_client_chunk_size = int(
+            registry.settings.get('queue_client_chunk_size', 1024)
+        )
+        queue_client_batch_size = int(
+            registry.settings.get('queue_client_batch_size', 1024)
+        )
+        if is_queue_server:
+            if not queue_type or queue_type == 'Simple':
```

Comment:
What happens if there is a queue_type that's not `Simple`?

Reply:
Yes, it wouldn't work.  This is temporary.  

In the next PR we'll add a redis queue.  Then 'Simple' queue will be the default.  If, for example, the queue in the config doesn't exists or the config says 'Redis' but redis isn't installed or running. 

Other queues could be AWS SQS or even an ES queue.  But I only have a redis one functioning.  The goal is that this pattern will allow us to plug in other queues as needed.

Reply:
An additional thing here is that all processes try to add an indexer to their registry.  But only the primary, viz, and region indexers should, not the 6 encoded application process.  I have a fix for that in the indexer.py includeme in a different PR, so this code is a little off.